lib/languageclient.js

```diff
@@ -129,6 +134,10 @@ export class LanguageClientConnection {
     this._sendNotification('textDocument/didClose', params);
   }
 
+  willSaveTextDocument(params: WillSaveTextDocumentParams): void {
```

Comment:
Should probably add a Public doc comment for consistency with the other methods.

Reply:
Will do, I'll add comments on others that are missing them too.